tests/test_eigen.py

```diff
@@ -251,14 +251,14 @@ def array_copy_but_one(a, r, c, v):
 def test_eigen_return_references():
     """Tests various ways of returning references and non-referencing copies"""
 
-    master = np.ones((10, 10))
```

Comment:
What do you think of `primary` as the new name?

`main` is a heavily used name for other things (C++ `main()`, Python `if __name__ == "__main__"`).

Reply:
Good point. I'll change that around!